Status: in BTS
Author: aurel32
Note: libtool update is necessary (libtoolize -c -f && aclocal-1.9 && autoconf && automake-1.9)

diff -u beep-media-player-0.9.7/debian/control beep-media-player-0.9.7/debian/control
--- beep-media-player-0.9.7/debian/control
+++ beep-media-player-0.9.7/debian/control
@@ -3,7 +3,7 @@
 Priority: optional
 Maintainer: Michiel Sikkes <michiel@eyesopened.nl>
 Uploaders: Paul van Tilburg <paulvt@debian.org>
-Build-Depends: debhelper (>= 4.0.0), libgtk2.0-dev, libvorbis-dev, libesd0-dev, libgnomevfs2-dev, libasound2-dev, libid3-3.8.3-dev, libglade2-dev
+Build-Depends: debhelper (>= 4.0.0), libgtk2.0-dev, libvorbis-dev, libesd0-dev, libgnomevfs2-dev, libasound2-dev [!kfreebsd-i386], libid3-3.8.3-dev, libglade2-dev
 Standards-Version: 3.6.1
 
 Package: beep-media-player

